resources/options_lookup.tsv

```diff
@@ -1222,275 +1222,275 @@ HVAC System Is Shared	Neither Shared
+		ResidentialAirflow																																													
 		ResidentialHVACSizing																																													
-HVAC System Heating Fuel Oil	"Oil Boiler, 90% AFUE"	ResidentialHVACBoiler	fuel_type=oil	"system_type=hot water, condensing"	afue=0.9	oat_reset_enabled=false	oat_high=0.0	oat_low=0.0	oat_hwst_high=0.0	oat_hwst_low=0.0	design_temp=180.0	capacity=autosize	dse=NA																																		
-		ResidentialAirflow	has_hvac_flue=false																																												
+HVAC System Heating Fuel Oil	"Oil Boiler, 90% AFUE"	ResidentialHVACBoiler	fuel_type=oil	"system_type=hot water, condensing"	afue=0.9	oat_reset_enabled=false	oat_high=0.0	oat_low=0.0	oat_hwst_high=0.0	oat_hwst_low=0.0	design_temp=180.0	capacity=autosize	dse=NA	has_hvac_flue=false																																	
+		ResidentialAirflow																																													
 		ResidentialHVACSizing																																													
-HVAC System Heating Fuel Oil	"Oil Boiler, 95% AFUE"	ResidentialHVACBoiler	fuel_type=oil	"system_type=hot water, condensing"	afue=0.95	oat_reset_enabled=false	oat_high=0.0	oat_low=0.0	oat_hwst_high=0.0	oat_hwst_low=0.0	design_temp=180.0	capacity=autosize																																			
-		ResidentialAirflow	has_hvac_flue=false																																												
+HVAC System Heating Fuel Oil	"Oil Boiler, 95% AFUE"	ResidentialHVACBoiler	fuel_type=oil	"system_type=hot water, condensing"	afue=0.95	oat_reset_enabled=false	oat_high=0.0	oat_low=0.0	oat_hwst_high=0.0	oat_hwst_low=0.0	design_temp=180.0	capacity=autosize	dse=NA	has_hvac_flue=false																																	
+		ResidentialAirflow																																													
 		ResidentialHVACSizing																																													
-HVAC System Heating Fuel Oil	Oil Boiler	ResidentialHVACBoiler	fuel_type=oil	"system_type=hot water, forced draft"	afue=0.76	oat_reset_enabled=false	oat_high=0.0	oat_low=0.0	oat_hwst_high=0.0	oat_hwst_low=0.0	design_temp=180.0	capacity=autosize	dse=NA																																		
-		ResidentialAirflow	has_hvac_flue=true																																												
+HVAC System Heating Fuel Oil	Oil Boiler	ResidentialHVACBoiler	fuel_type=oil	"system_type=hot water, forced draft"	afue=0.76	oat_reset_enabled=false	oat_high=0.0	oat_low=0.0	oat_hwst_high=0.0	oat_hwst_low=0.0	design_temp=180.0	capacity=autosize	dse=NA	has_hvac_flue=true																																	
+		ResidentialAirflow																																													
 		ResidentialHVACSizing																																													
 HVAC System Heating Wood	None																																														
 HVAC System Heating Wood	Wood Stove	ResidentialHVACUnitHeater	fuel_type=wood	efficiency=0.6	fan_power=0	airflow_rate=0	capacity=autosize																																								

```

Comment:
Move flue argument from airflow to `ResidentialHVACUnitHeater`?

Reply:
...This is meant to reference the `HVAC System Heating Wood` option

Reply:
Oops, missed one. Good catch.